lua/obsidian/note.lua

```diff
@@ -408,4 +408,17 @@ note.save = function(self, path, insert_frontmatter)
   return lines
 end
 
+--Jump to a series of nested headers
+--Used with :ObsidianFollowLink to follow links in the form [[note#header]]
+--Also accepts [[#header]] or [[note#header1#header2]] to indicate hierarchy
+--
+---@param headers table
+note.jump_to_header = function(headers)
```

Comment:
I think it would make more sense to put this into `lua/obsidian/util.lua`. The functions in this file are meant to be instance methods on `Note` objects.

Reply:
On second thought I agree, have pushed a change moving this to `util.lua`

Reply:
Thanks! Can you fix the issue with new note completion and then we'll get this merged?